src/libs/actions/ReimbursementAccount/navigation.js

```diff
@@ -7,7 +7,7 @@ import ROUTES from '@src/ROUTES';
  * Navigate to a specific step in the VBA flow
  *
  * @param {String} stepID
- * @param {Object} newAchData
+ * @param {Object} [newAchData]
```

Comment:
Has that actually been made optional? dont see a default value in the function params

Reply:
actually `newAchData` is never passed in whenever this function is used across the app, but this didn't come up in js files. I think we didn't want to remove it due to not having enough context on it, but typescript was not happy ;)

Reply:
Ah so if its still not used anywhere, should we just remove it here?

Reply:
sure, removed it 👍🏻